examples/web/index.html

```diff
@@ -8,13 +8,13 @@
   <!-- iOS meta tags & icons -->
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
-  <meta name="apple-mobile-web-app-title" content="Flame Example">
+  <meta name="apple-mobile-web-app-title" content="Flame Examples">
   <link rel="apple-touch-icon" href="icons/Icon-192.png">
 
   <!-- Favicon -->
   <link rel="icon" type="image/png" href="favicon.png"/>
 
-  <title>Flame Example</title>
+  <title>Flame Examples</title>
```

Comment:
This was just inconsistent. Funnily, the page loads with this name and then the JS kicks in and changes it to the real name (which meant adding this `s`)